sql/parser/sql.y

```diff
@@ -32,11 +32,11 @@ import "strings"
   str            string
   strs           []string
   qname          QualifiedName
-  qnames         []QualifiedName
+  qnames         QualifiedNames
   stmt           Statement
   stmts          []Statement
-  tableDef       TableDef
-  tableDefs      []TableDef
+  tblDef         TableDef
```

Comment:
I renamed these to be consistent with `tbl{Expr,Exprs}`. 

This is part of the semantic action code. It is completely different upstream (i.e. written in C).